src/components/probe/index.ts

```diff
@@ -23,23 +23,26 @@
  **********************************************************************************/
 
 import {
-  PROBE_STATUS_PROCESSED,
   PROBE_RESPONSE_RECEIVED,
   PROBE_RESPONSE_VALIDATED,
+  PROBE_ALERTS_READY,
+  PROBE_LOGS_BUILT,
 } from '../../constants/event-emitter'
+import { LogObject } from '../../interfaces/logs'
 import { Notification } from '../../interfaces/notification'
 import { Probe } from '../../interfaces/probe'
 import { AxiosResponseWithExtraData } from '../../interfaces/request'
 import { ValidateResponse } from '../../plugins/validate-response'
 import { getEventEmitter } from '../../utils/events'
-import { printProbeLog, probeLog, setAlert } from '../logger'
-import { processProbeStatus } from '../notification/process-server-status'
+import { probeBuildLog, setAlert } from '../logger'
+import { processThresholds } from '../notification/process-server-status'
 import { probing } from './probing'
 
 const em = getEventEmitter()
 
 let validatedRes: ValidateResponse[] = []
 
+// 2. Responses have been processed and validated
```

Comment:
why do we have number `2` in this comment?
Where is number `1` etc?

Reply:
removed.